### ISSUE TITLE
Documentation - Memory usage with default of Browser.RetainLogs == true

### ISSUE THREAD

**Issue by rjk** [SimpleBrowserDotNet/SimpleBrowser#199]
I found that when doing lots of browsing with a default Browser object the memory usage creeps up and up (in my case blowing 4GB within a few hours). The culprit is that `Browser.RetainLogs` is set to true by default so it keeps lots of history. It would be good to point out in the tutorial that if you're doing bulk web scraping you'll want to set `browser.RetainLogs = false` to avoid this problem.

**Comment by kevingy** [comment]
I agree. Personally, I always turn off logs. That said, I also create a browser, make a few calls, then destroy the browser. I don't have them lingering for long - typically, less than 2 minutes.

We have another issue that deals with duplication of history and logs. I will try to find some time to look at these soon.
